data/gegno/gegno.txt

```diff
@@ -14,24 +14,22 @@
 phrase "gegno vi"
 	word
 		"" 4
-		"Beuci"
-		"Dueyu"
-		"Tie"
-		"Efta"
-		"Gye"
-		"Kyhe"
-		"Efe"
-		"Eys"
-		"Imnn"
-		"Yx"
-		"Isyp"
-		"Tyyn"
-		"Hy"
-		"Eysyp"
-		"Iidt"
-		"Ych"
-	word
-		" "
+		"Beuci "
+		"Dueyu "
+		"Tie "
+		"Efta "
+		"Gye "
+		"Kyhe "
+		"Efe "
+		"Eys "
+		"Imnn "
+		"Yx "
+		"Isyp "
+		"Tyyn "
+		"Hy "
+		"Eysyp "
+		"Iidt "
+		"Ych "
```

Comment:
Sorry, what's the reasoning behind this change? It might marginally increase performance, but it'll make the datafile a lot harder to edit.

Reply:
Wait, I only saw the reasoning now lol